dashboard/app/handler.go

```diff
@@ -36,6 +36,7 @@ func handleContext(fn contextHandler) http.Handler {
 		if !throttleRequest(c, w, r) {
 			return
 		}
+		defer backpressureRobots(c, r)
```

Comment:
This misses `()` at the end. In the current form it's a no-op.

Reply:
Thanks.